ios/Podfile

```diff
@@ -5,5 +5,7 @@ target 'Pangea' do
   pod 'RSKImageCropper', '2.0.0'
   pod 'QBImagePickerController', '3.4.0'
 
+  pod 'react-native-version-number', :path => '../node_modules/react-native-version-number'
```

Comment:
I don't think we need it in case we added the library manually to `Libraries` folder. And this line cause some issues (at least on my side) after running `pod install`, however works fine if I remove that line.